packages/backend/src/config/app-env-vars/formsg.ts

```diff
@@ -0,0 +1,14 @@
+export const formSgConfig = Object.freeze({
+  apiKeys: {
+    prod: process.env.FORMSG_API_KEY,
```

Comment:
Getting more and more clunky to manage these env vars. Thinking to move to a different approach where we query these secrets from our DB instead (see this abandoned [PR](https://github.com/opengovsg/plumber/pull/616) for example - I created an `app_metadata` table)